code/modules/mob/living/carbon/xenomorph/Facehuggers.dm

```diff
@@ -92,6 +92,11 @@
 	return FALSE // Else you can't pick.
 
 /obj/item/clothing/mask/facehugger/attack(mob/M, mob/user)
+	user.visible_message("<span class='warning'>\ [user] attempts to plant [src] on [M]'s face!</span>", \
+	"<span class='warning'>You attempt to plant [src] on [M]'s face!</span>")
+	if(ishuman(M)) //Delay for humans only, not monkeys and shit
+		if(!do_after(user, 5, TRUE, 5, BUSY_ICON_HOSTILE))
+			return
 	if(CanHug(M))
```

Comment:
This could go before, as an early return. Better than adding a delay before informing them it can't be attached.